### ISSUE TITLE
logrotate is not reload/restarting httpd which leaves a stale open file

### ISSUE THREAD

**Issue by pbiering** [ConSol/omd#137]
found via stale open file check:

```
perl      1516938                      mon1    3w      REG                8,2     20527     0   12587824 /opt/omd/sites/mon1/var/log/thruk.log.1 (deleted)
```

It's a child process of specific `httpd` instance:

```
           |-httpd(1913505,mon1)-+-httpd(1516927)
           |                        |-httpd(1516932)
           |                        |-httpd(1517011)
           |                        |-httpd(1517017)
           |                        |-httpd(1913509)---perl(1516938) <-!!!
           |                        |-httpd(1913513)
           |                        `-perl(1913507)
```
and turned out to be the Fast-CGI process of `thruk`

```
mon1  1516938  0.0  0.8 180424 71164 ?        S    Aug30   0:02 perl /omd/sites/mon1/share/thruk/script/thruk_fastcgi.pl
```
Workaround:
```
service omd restart
```

but potentially the generated logrotate config of `omd` can cover this itself automagically

**Comment by sni** [comment]
Apaches post rotate script reloads the httpd here: https://github.com/ConSol/omd/blob/labs/packages/apache/skel/etc/logrotate.d/zzz-apache.
We recently renamed the script, so it might still be called `apache` in your case.
In the past, we had some cases were logrotate thought a rotation is not required. Details could be found in `var/log/logrotate.log`.

**Comment by pbiering** [comment]
@sni: in the current RPM there is no skeleton file related to apache, also the `logrotate.log` misses any reference:

```
# rpm -ql omd-4.60-labs-edition-el8-1.x86_64 | grep logrotate | grep apache
(no output)

# grep httpd /opt/omd/sites/*/etc/logrotate.d/*
(no output)

# grep httpd  /opt/omd/sites/*/var/log/logrotate.log
(no output)
```

Is there a RPM packaging issue and the skeleton file was missed?

BTW: the link is broken.

**Comment by sni** [comment]
this has been fixed with 8722ec091068a5833444b0f26c91f39f4daf3b8d
